mono/mini/debugger-agent.c

```diff
@@ -720,6 +722,8 @@ static void process_breakpoint_events (void *_evts, MonoMethod *method, MonoCont
 static int ss_create_init_args (SingleStepReq *ss_req, SingleStepArgs *args);
 static void ss_args_destroy (SingleStepArgs *ss_args);
 
+static GENERATE_GET_CLASS_WITH_CACHE (fixed_buffer, "System.Runtime.CompilerServices", "FixedBufferAttribute")
```

Comment:
maybe `GENERATE_TRY_GET_CLASS_WITH_CACHE` is better in case the `FixedBufferAttribute` class was removed by the mono linker?

Reply:
Done!